aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/util/ResourceModelHelper.java

```diff
@@ -48,6 +48,10 @@ public static boolean isSqlServer(final ResourceModel model) {
         return engine == null || engine.contains(SQLSERVER_ENGINE);
     }
 
+    public static boolean isMySQL(final ResourceModel model) {
+        final String engine = model.getEngine();
+        return engine != null && engine.startsWith(MYSQL_ENGINE_PREFIX);
```

Comment:
Are `aurora-mysql` and `aurora` engines considered MYSQL engine? If yes, we need to conisder that too

Reply:
The link you have posted also mentions Oracle DB instances. Shall we use this parameter in that case too?